src/pages/HomePage.jsx

```diff
@@ -0,0 +1,9 @@
+import { MoviesList } from "./MoviesList/MoviesList";
+
+export const HomePage = () => {
+  return (
+    <>
+      <MoviesList />
+    </>
+  );
+};
```

Comment:
Hmm, maybe this HomePage component is not neccessary, since you're only mounting one thing in it? 👀